tasks/dns_records.yml

```diff
@@ -14,4 +14,15 @@
       - "{{ dkim_dns_record.content | string | b64decode }}"
       - "################################################################################################"
 
+- local_action:
+    module: copy
+    content: |
+          ################################################################################################
+          DNS record to add for zone {{ dkim_domain }}:
+          {{ dkim_dns_record.content | string | b64decode }}
+          ################################################################################################
+    dest: dns_record.{{ dkim_domain }}
```

Comment:
Where those files will be exactly located?

Reply:
In the directory in which you started the ansiblw script. That is the only place I knew for sure that existed. I thought about creating an output dir, etc, but this seemed most simple. And by default nothing will be created of course. You have to request it.

Reply:
Files will be named: dns_record.example1.com,  dns_record.example2.org

Reply:
I use something like this when saving password. Could be nice for dns records too:
```
percona_local_passdb: "{{ local_passdb | default('~/.ansible_percona_passwords') }}"
percona_pass_len: 24
percona_pass_complexity: "ascii_letters,digits" # ascii_letters,digits,hexdigits,punctuation
percona_root_password: "{{ lookup('password', [percona_local_passdb, '/', inventory_hostname, '/', 'percona.root', ' chars={{ percona_pass_complexity }} length={{ percona_pass_len }}']|join) }}"
```